formspree/forms/models.py

```diff
@@ -198,10 +198,13 @@ def send(self, data, keys, referrer):
         # check if the forms are over the counter and the user is not upgraded
         overlimit = False
         monthly_counter = self.get_monthly_counter()
-        if monthly_counter > settings.MONTHLY_SUBMISSIONS_LIMIT and not self.upgraded:
+        monthly_limit = settings.MONTHLY_SUBMISSIONS_LIMIT if \
+                self.id > settings.FORM_LIMIT_DECREASE_ACTIVATION_SEQUENCE \
+                else 1000
```

Comment:
Not a fan of this magic number 1000. We should at least put a constant variable at file scope called "GRANDFATHER_MONTHLY_LIMIT" or something